### DISCUSSION TITLE
Can't access my firefly wallet

### DISCUSSION THREAD

**Discussion by Cayniz** [iotaledger/firefly#8626] [Help & Support]
Hello! I was trying to access my Firefly wallet but I received the following error timestamp . I'm trying to figure out how to troubleshoot this issue. Has anyone else ran into this problem? hope to hear from someone soon!

Error Timestamp:

Sat, 16 Nov 2024 01:16:23 GMT
wallet: `local time 1731719783 doesn't match the time of the latest milestone timestamp: 1731719345`

**Comment by DrOyecomova** [comment]
The same thing has been happening to me for a few days as well.

**Reply by kilianhln** [reply]
> > The same thing has been happening to me for a few days as well.
> 
> Same here blackhathacker110 on telegram helped me fix it

@DrOyecomova NEVER share your seed with some random Telegram users.

**Comment by begonaalvarezd** [comment]
The error means that your machine has an incorrect system clock, make sure your clock is correct my syncing it or setting it to automatic
